src/zkevm_specs/evm/table.py

```diff
@@ -179,9 +190,9 @@ def write_with_reversion(self) -> bool:
 
 
 class AccountFieldTag(IntEnum):
-    Nonce = auto()
-    Balance = auto()
-    CodeHash = auto()
+    Nonce = 1
```

Comment:
Did you change these to powers of 2 so that it will be easier to select between them once we switch to using binary columns for the tag and field tag?

Reply:
Ah, I didn't think about that.  The reason why I changed these to powers of 2 is because currently the MTP circuit is using boolean columns for each one of the possible account fields.  For the lookup expression it looks simpler if those 4 binary columns are merged into a single value, and I thought using powers of 2 would never mix the columns (it's just a binary encoding of the 4 columns).  Nevertheless the MPT circuit probably constraints that at most one of these 4 binary columns is 1, so using `1,2,3,4` would work as well I guess.

Reply:
Makes sense.

My guess is that we will want these to be 1, 2, 3, 4, ... in a single column the state circuit, but I'm not positive. I'm fine with this change, since it'll be very easy to go back if needed when we add the MPT constraints in the state circuit.

Reply:
I've decided to revert this change because after thinking about it, I don't find a compelling reason to change these values, so I'll leave them untouched.